connectivity/drivers/lora/COMPONENT_SX126X/SX126X_LoRaRadio.cpp

```diff
@@ -725,6 +737,12 @@ uint8_t SX126X_LoRaRadio::get_frequency_support(void)
         return (MATCHING_FREQ_868);
     }
 }
+#else
+uint8_t SX126X_LoRaRadio::get_frequency_support(void)
```

Comment:
Same as the comment above, this can be simplified to have one function that do different things based on the configuration (rather  than 2 separate functions).